components/Accordion.js

```diff
@@ -0,0 +1,39 @@
+
+import Chevron from "./Chevron.js";
+import React, { useState, useRef } from 'react';
+import styles from "../styles/Accordion.module.css";
+
+export default function Accordion({accordionTitle}, {accordionContent}) {
```

Comment:
Should write props like this for simplicity: ({ accordionTitle, accordionContent })